code/flist_messenger/flist_account.cpp

```
@@ -25,13 +25,17 @@ void FAccount::loginSslErrors( QList<QSslError> sslerrors )
 
 void FAccount::onLoginError(QString error_id, QString error_message)
 {
-	debugMessage("account->loginHttps() error!");
-	emit loginError(this, "Login Error", QString("%1 (%2)").arg(error_message).arg(error_id));
+    debugMessage("account->loginHttps() error!");
+    if (error_message.compare("Password mismatch.")) {
```

Do we need to hide the error ID for errors that aren't password failures?